packages/app-mobile/root.tsx

```diff
@@ -64,7 +64,7 @@ const { SearchScreen } = require('./components/screens/search.js');
 const { OneDriveLoginScreen } = require('./components/screens/onedrive-login.js');
 import EncryptionConfigScreen from './components/screens/encryption-config';
 const { DropboxLoginScreen } = require('./components/screens/dropbox-login.js');
-const { MenuContext } = require('react-native-popup-menu');
+const { MenuProvider } = require('react-native-popup-menu');
```

Comment:
```suggestion
import { MenuProvider } from 'react-native-popup-menu';
```
(Very minor/optional) Consider using the typed version of `react-native-popup-menu` here!